WP_CLI_CS/ruleset.xml

```diff
@@ -79,9 +79,23 @@
 		<exclude name="Universal.Operators.DisallowShortTernary.Found"/>
 
 		<!-- Allow filesystem operations because WordPress APIs may not be available -->
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_chgrp" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_chmod" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_chown" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fclose" />
 		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_file_put_contents" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fopen" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fputs" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fread" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fsockopen" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fwrite" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_is_writable" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_is_writeable" />
 		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_mkdir" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_pfsockopen" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_readfile" />
 		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_rmdir" />
+		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_touch" />
```

Comment:
This whole list should be removed (maintenance overhead) as the `exclude` for `file_system_operations` on line 148 does the exact same thing already (just more efficiently).